From 49d12a14482ce9a193c049a5ccdd2fed90b0bdbe Mon Sep 17 00:00:00 2001
From: Valentin Herrmann <me@valentin-herrmann.de>
Date: Sat, 11 Nov 2023 12:49:02 +0100
Subject: [PATCH] fix: repl: Make whitespace case more precise

---
 elisp/geiser-repl.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/elisp/geiser-repl.el b/elisp/geiser-repl.el
index dfdc26b..dbf767a 100644
--- a/elisp/geiser-repl.el
+++ b/elisp/geiser-repl.el
@@ -934,7 +934,8 @@ paredit."
         (compilation-forget-errors))
       (geiser-repl--prepare-send)
       (comint-send-input)
-      (when (string-match "^\\s-*$" intxt)
+      ;; match if `intxt' is lines of whitespace
+      (when (string-match "\\`\\(\\s-\\|\n\\)*\\'" intxt)
         (comint-send-string proc (geiser-eval--scheme-str '(:ge no-values)))
         (comint-send-string proc "\n")))))
 
-- 
2.40.1

